common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -16,57 +16,61 @@ type StaticType = { a: string; b: string };
 type IndexSignatureType = { [key: string]: string };
 
 const nestedObj: NestedIndexProps = { nested: { a: "hello" } };
-nestedObj.nested.a; // ok: Accessing nested property 'a', 'a' is defined
+nestedObj.nested.a; // ok: Accessing nested index property 'a' without requiring a particular result
 nestedObj.nested.a.length; // defect: Accessing length of a nested undefined property
 
 const a = "a";
 const b = "b";
 const c = "c";
 const someObjWithStaticType: StaticType = { a: "hello", b: "goodbye" };
-someObjWithStaticType.a; // ok: Accessing property 'a', 'a' is defined
-someObjWithStaticType.a.length; // ok: Accessing length of string property 'a', 'a' is a string
+someObjWithStaticType.a; // ok: Accessing string property 'a'
+someObjWithStaticType.a.length; // ok: Accessing length of string property 'a'
 someObjWithStaticType["a"]; // ok: Accessing property 'a' using bracket notation
 someObjWithStaticType["a"].length; // ok: Accessing length of string property 'a' using bracket notation
 someObjWithStaticType[a].length; // ok: Accessing length of string property 'a' using bracket notation
 someObjWithStaticType[b].length; // ok: Accessing length of string property 'b' using bracket notation
-someObjWithStaticType[c].length; // defect: Accessing length of undefined property 'c' using bracket notation
-const aExpectingStringFromStaticType: string = someObjWithStaticType.a; // ok: Assigning string property to a strict string variable, 'a' is defined
+someObjWithStaticType[c].length; // defect: Accessing length of missing property 'c' using bracket notation
```

Comment:
No need for lint rule to flag this. This should be a tsc error already. Lint rule should be limited to indexed type accesses. That could be the reason that none of the cases with `StaticType` are defects.
If you want to keep this line, then add a comment before it with `// @ts-expect-error`.

Reply:
Sounds good, I was thinking similar to this, I will remove it since its supposed to be caught by tsc